src/utils/whitelist-cookies.js

```diff
@@ -1,5 +1,13 @@
+/**
+ * TODO: document how to put this into `cypress/support/index.js` as suggested
+ * by the auth0 docs.
+ */
 Cypress.Cookies.defaults({
   preserve: [
-    Cypress.env('auth0SessionCookieName')
+    Cypress.env('auth0SessionCookieName'),
+    // We assume that the cookie is never splitted into more than 10 parts
+    ...Array(10)
+      .fill(1)
+      .map((x, y) => `${Cypress.env('auth0SessionCookieName')}.${x + y}`),
```

Comment:
Would the following be more readable here?

```js
.map((value, i) => `${Cypress.env('auth0SessionCookieName')}.${value + i}`),
```

Or something like that?

Reply:
Yes, is more readable, see https://github.com/sir-dunxalot/cypress-nextjs-auth0/pull/44/commits/f10bc5a4052f922e72bb1222f8897328549b7e7e, please.